assignments/arrays.js

```diff
@@ -63,32 +63,54 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let car33 = 0
 
+console.log(`Car 33 is a ${inventory[32].2011} ${inventory[32].Jeep} ${inventory[32].Wrangler}` );
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[inventory.length-1];
+console.log(lastCar.Lincoln, lastCar.Town-car);
```

Comment:
Same review here. You should avoid referencing the value on the dot notation. Always reference the key.